app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+  def days_of_week(date)
+    date.beginning_of_week..date.end_of_week
```

Comment:
You can do: `date.all_week`
```suggestion
    date.all_week
```

Reply:
indeed, I completely forgot about these. => no need for the helper methods at all!